src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -2131,6 +2162,7 @@ private static bool IsUserDefinedTrueOrFalse(BoundUnaryOperator @operator)
         {
             if (lowestBoundNode is BoundExpression boundExpr)
             {
+                boundExpr = AdjustLowestBoundNodeForFunctionPointerConversion(boundExpr);
```

Comment:
>boundExpr = AdjustLowestBoundNodeForFunctionPointerConversion(boundExpr); [](start = 16, length = 73)

It looks like based on our offline conversation this special case is not needed too.

Reply:
Yep, I've already removed it locally.

---
In reply to: [430767234](https://github.com/dotnet/roslyn/pull/44436#discussion_r430767234) [](ancestors = 430767234)